mesonbuild/environment.py

```diff
@@ -260,7 +260,7 @@ def detect_system():
     system = platform.system().lower()
     if system.startswith('cygwin'):
         return 'cygwin'
-    return system
+    return system.lower()
```

Comment:
But system should already be in lower case, it is created three lines above with `.lower()`.

Reply:
so it is, I'll drop that